jobspec/parse.go

```diff
@@ -477,3 +513,35 @@ func parseUpdate(result *structs.UpdateStrategy, obj *hclobj.Object) error {
 	}
 	return nil
 }
+
+func toDuration(value interface{}) (time.Duration, error) {
```

Comment:
Should we instead just require that the input be a string, and be parseable by Go's `time.ParseDuration`?